### PR TITLE
Readme tidyup

### PR DESCRIPTION
**Pull request by wilsondennison**
Readme tidyup

### CODE REVIEW

**Comment by McDonnellDean** [comment]
Checked.

@mcollina CANIHAZPULLACCESZ :dancer:

File: README.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 
-Artistic License 2.0
+# Amazon Machine Image Container
+Manages the creation, deployment, linking and unlinking of an Amazon Machine Image for __nscale__. The
+aws-ami-container is used internally by __nscale__, see __nscale__'s [documentation] for information on how
+to deploy using Amazon Web Services as the target platform.
+
+
+## License
+Copyright (c) 2015 [nearForm] and other contributors
```
Comment (mcollina) [inline README.md]: Copyright year should be 2014-2015.

Comment (McDonnellDean) [inline README.md]: @wilsondennison to fix this all you need to do is add another commit to your branch and sync it to github, the PR will pick it up automatically. I will change the ones we merged so just this and the ELB container is all you need to do.

**Comment by wilsondennison** [comment]
@McDonnellDean  done :)

**Pull request merged**